WordPress/gutenberg
# Icons: Remove width / height attributes from svg element

~~Fix: #43745~~
Related: #40315

## What?
This PR removes the `width` / `height` attributes from the svg element in the `@wordpress/icons` icon.

## Why?
As tested in #43745, the `iconSize` prop is not applied when an icon with `width` / `height` attributes is specified in the `Button` component.
_@edit_: This issue is fixed by #43821.

We could not find a clear reason why the `Icons` component could resize these icons correctly, but at least the svg icons themselves don't need `width` / `height`, and removing these attributes gave the expected results.

## How?
Simply removed attributes.

## Testing Instructions
- In both the trunk and this branch, use code blocks to execute the following code:

```javascript
import { useBlockProps } from '@wordpress/block-editor';
import * as Icons from '@wordpress/icons';
import { Button, Flex } from '@wordpress/components';

const { Icon, ...availableIcons } = Icons;

export default function CodeEdit() {
	const blockProps = useBlockProps();
	return (
		<div { ...blockProps }>
			<Flex
				wrap="wrap"
				justify="flex-start"
				style={ { textAlign: 'center', fontSize: '8px' } }
			>
				{ Object.entries( availableIcons ).map( ( [ name, icon ] ) => {
					return (
						<div key={ name } style={ { width: '5%' } }>
							<Button icon={ icon } iconSize={ 40 } />
						</div>
					);
				} ) }
			</Flex>
		</div>
	);
}
```

- Insert the code block in the editor and confirm that the icon list appears.
- Compare the build results between trunk and this branch and confirm that the icons circled in red are displayed in the correct size in this branch.
